crm_compassion/models/interaction_resume.py

```diff
@@ -162,3 +163,12 @@ def open_object(self):
         """
         self.ensure_one()
         return True
+
+    def _compute_has_attachment(self):
+        """
+        Check in each row if e-mail (outgoing) or message (incoming) contains
+        at least 1 attachment.
+        """
+        for row in self:
+            row.has_attachment = row.email_id.attachment_ids or \
+                                 row.message_id.attachment_ids
```

Comment:
```suggestion
                row.message_id.attachment_ids
```
Only one more tab indent is needed after \